hierarqcal/utils.py

```diff
@@ -144,10 +147,81 @@ def plot_motif(
             )
             ax.add_patch(arrow)
         else:
-            # TODO hyper graph visual (rubber band)
-            raise NotImplementedError(
-                f"Plotting {len(edge)}-ary edges are not implemented yet"
-            )
+            end_points = []
+            for i in range(len(edge)):
```

Comment:
Thanks for adding the colors! Just as I mentioned this morning, the node colors has specific meaning, so I originally meant that it's the edge that should be broken up into three color parts. We can just color those loops that gets added around the nodes to indicate, start, middle, end

Reply:
Added.